server/Hubs/GameHub.cs

```diff
@@ -0,0 +1,13 @@
+using Microsoft.AspNetCore.SignalR;
+using System.Threading.Tasks;
+
+namespace Api.Hubs
+{
+    public class GameHub : Hub
+    {
+        public async Task SendMessage()
```

Comment:
Does the name of the method determines which event triggers it ?

Reply:
Yes, you call `hubConnection.invoke('SendMessage')` on the client to trigger this.